### ISSUE TITLE
It would be very kind that if you could provide the pre-trained model for CWAT and MTD.

### ISSUE THREAD

**Issue by beautyremain** [7eu7d7/RobustDet#10]
As you've mentioned in readme.md, both CWAT and MTD have no official code for training or evaluation. I've tried to reproduce CWAT and MTD by the train_adv.py with exactly the same setting as required, but it is not working fine (under cls-20 attack, 0.09 for CWAT and 0.10 for MTD). I would be really grateful if you could provide pre-trained models for CWAT and MTD. And hopefully, it would help me to finish my work.

**Comment by xmasyu** [comment]
Hello, may I ask if you can now obtain the results of the mtd and cwat papers on your end

**Comment by beautyremain** [comment]
Thank you so much for your reply. Not yet, and since we found some super abnormal phenomenon when training epoch is extremely large, we halted the research about this topic.  Still, due to my attempting reproduction on CWAT and MTD was pretty bad, I'm still hoping to get the correct code  to determine my previous problem.

**Comment by xmasyu** [comment]
I think there is a chance in the training of mtd and cwat, because its loss has been fluctuating significantly during the training process. I trained the weight of cwat and its test results were relatively normal. So I feel like if you want to get the results on mtd and cwat in the paper, you need to train multiple times to try it out

**Comment by Mr-Teal** [comment]
Hi! Are you able to run the evaluation process with the pretrained RobustDet model they provided?

**Comment by beautyremain** [comment]
> Hi! Are you able to run the evaluation process with the pretrained RobustDet model they provided?

Yes, the pretrained RobustDet model works well without any modification.

**Comment by Mr-Teal** [comment]
> > Hi! Are you able to run the evaluation process with the pretrained RobustDet model they provided?
> 
> Yes, the pretrained RobustDet model works well without any modification.

Which system do u evaluate the model on?
At first I could not run the model properly on my Linux server, but when I ran it on my Windows, it worked.

**Comment by beautyremain** [comment]
> > > Hi! Are you able to run the evaluation process with the pretrained RobustDet model they provided?
> > 
> > 
> > Yes, the pretrained RobustDet model works well without any modification.
> 
> Which system do u evaluate the model on? At first I could not run the model properly on my Linux server, but when I ran it on my Windows, it worked.

on linux, I remember some debugging process but im not quite sure what ive done, but it worked eventually.
